spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerTable.java

```diff
@@ -178,4 +178,10 @@ public String name() {
   public ScanBuilder newScanBuilder(CaseInsensitiveStringMap options) {
     return new SpannerScanBuilder(options);
   }
+
+  @Override
+  public WriteBuilder newWriteBuilder(LogicalWriteInfo info) {
+    // Unsupported!
+    throw new RuntimeException("writes are not supported in the Spark Spanner Connector");
```

Comment:
Please use SpannerConnectorException. 

For this case, we can add a error code in SpannerErrorCode like WRITE_NOT_SUPPORT.

Reply:
Did we merge that PR already?

Reply:
I've set SpannerConnectorException, thanks @halio-g! I am going to add the error code shortly too.

Reply:
Error code set too, thanks @halio-g!